iceoryx_hoofs/source/posix_wrapper/message_queue.cpp

```diff
@@ -204,8 +204,8 @@ cxx::expected<std::string, IpcChannelError> MessageQueue::receive() const noexce
     return cxx::success<std::string>(std::string(&(message[0])));
 }
 
-cxx::expected<int32_t, IpcChannelError> MessageQueue::open(const IpcChannelName_t& name,
-                                                           const IpcChannelSide channelSide) noexcept
+cxx::expected<mqd_t, IpcChannelError> MessageQueue::open(const IpcChannelName_t& name,
```

Comment:
Hint: `mqd_t` is not a file descriptor and therefore not convertable to an `int32_t` this worked just by accident.